msg/GimbalCmd.msg

```diff
@@ -0,0 +1,2 @@
+int8 left_right # 'a' -> -1 = left, 'd' -> 1 = right, 0 = noop
```

Comment:
Doesn't seem necessary to specify the controls in the interface. Just say that -1 means left, 1 means right, 0 is nothing